sections/interfaces.js

```diff
@@ -47,7 +47,7 @@ export default [
     type: "method",
     method: "then",
     example: ".then(onFulfilled, [onRejected])",
-    description: "Coerces the current query builder chain into a promise state, accepting the resolve and reject handlers as specified by the Promises/A+ spec. As stated in the spec, more than one call to the then method for the current query chain will resolve with the same value, in the order they were called; the query will not be executed multiple times.",
+    description: "Coerces the current query builder chain into a promise state, accepting the resolve and reject handlers as specified by the Promises/A+ spec. As stated in the spec, more than one call to the then method for the current query chain will resolve with the same value, in the order they were called; the query will not be executed multiple times. An additional option can be included if the table has triggers associated with it.",
```

Comment:
I don't think this is the right place.
Documentation for `insert`, `delete` and `update` operations should be changed, this is not a change to `then` operation.

Reply:
My bad. I found it. Didn't know exactly where everything was.